deployment/config/public-graphql-api-gateway/gateway.config.ts

```diff
@@ -0,0 +1,15 @@
+// @ts-expect-error not a dependency
+import { defineConfig } from '@graphql-hive/gateway';
+
+export const gatewayConfig = defineConfig({
```

Comment:
Any other plugins we should consider here? should we offload some of the plugins in graphql-api to the gw? (in a separate pr?)